blog/2023-11-05-databend-weekly-118.md

```diff
@@ -0,0 +1,110 @@
+
+In Databend, the `MERGE INTO` statement now supports distributed execution, which enables more efficient data integration capabilities. The physical plan for distributed execution is shown in the following diagram:
+
+![Image: Databend Copy Into](../static/img/blog/merge-into-physical-plan.png)
+
+Please [contact the Databend team](https://www.databend.com/contact-us) if you would like to learn more, or check out the resources below:
+
+- [PR #13151 | feat: support distributed merge_into](https://github.com/datafuselabs/databend/pull/13151)
+
+## Code Corner
+
+Discover some fascinating code snippets or projects that showcase our work or learning journey.
+
+### Incorporate Data Version Control into Your Analytics Workflow with LakeFS
+
+LakeFS provides a solution for sharing and collaborating on data processing by empowering object storage with Git-like operations. It offers a versioning scheme that provides logically consistent views of data, meaningful branch names, and commit messages for modern data workflows, as well as integrated solutions for data and document unification.
+
+Databend not only provides a distributed, elastic, and scalable query engine with high performance but also supports multi-version storage similar to Git. This allows for easy querying, cloning, and restoring of data at any point in time.
+
+By combining LakeFS's data versioning capabilities with Databend's Git-like data management capabilities, data integrity can be ensured, and efficient and highly available consistent views can be provided for data analysis workflows.
+
+If you are interested in learning more, please check out the resources below:
+
+- [Blog | Incorporate Data Version Control into Your Analytics Workflow](https://databend.rs/blog/2023-11-02-databend-with-lakefs)
+
+## Highlights
+
+We have also made these improvements to Databend that we hope you will find helpful:
+
+- Added support for JSON operators: `?`, `?|`, and `?&`.
+- Added support for permissions management on `STAGE` and `UDF`.
+- Added support for `flatten` table function.
+- Added support for creating external tables using URLs: `CREATE STAGE test_stage [URL=] 's3://load/files/'`.
+- Read the documentation [Example 2: Attaching Table in READ_ONLY Mode](https://databend.rs/doc/sql-commands/ddl/table/attach-table#example-2-attaching-table-in-read_only-mode) to learn how to use `ATTACH TABLE` in read-only mode.
+- Read the documentation [Conversion Functions](https://databend.rs/doc/sql-functions/conversion-functions/) to learn about type conversion functions and rules in Databend.
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+### Support for Reading Delta Lake
+
+Databend currently supports mounting and reading data from Iceberg tables through the Catalog, and the next planned target is Delta Lake.
```

Comment:
After successfully incorporating support for mounting and reading data from Iceberg tables through the Catalog, Databend is gearing up for adding compatibility with Delta Lake.